02.calendar/calendar.rb

```diff
@@ -0,0 +1,45 @@
+require 'date'
+require 'optparse'
+
+class Calender  
+  def initialize(year: Date.today.year, month: Date.today.month)
+    @year = year
+    @month = month
+  end
+  
+  def calender
+    start_month = Date.new(@year, @month, 1)  
+    end_month = Date.new(@year, @month, -1)
+    
+    puts "#{@month}月  #{@year}".center(20)
+    
+    days = ["日", "月", "火", "水", "木", "金", "土"]
+    days.each do |day|
+      print day.rjust(2)
+    end
+
+    print "\n"
+
+    blank = start_month.wday
+    (blank*3).times {print " "}
```

Comment:
```suggestion
    (blank * 3).times { print " " }
```

僕だったらこんなふうにスペースを入れますね。

あと、`blank`という変数名だと、「空白そのもの」に見えてしまうので、`blank_count`のように数値であることがわかるような変数名の方がベターです。
さらに言えば、この程度の短いロジックであれば、変数に入れなくても可読性はあまり変わらない気もします。

```ruby
(start_month.wday * 3).times { print " " }
```